overview.md

```diff
@@ -43,130 +43,173 @@ steps:
     condition: succeededOrFailed()
     inputs:
       testResultsFiles: test-results/results.xml
-  - task: PublishCodeCoverageResults@1
+  - task: PublishCodeCoverageResults@2
     inputs:
       codeCoverageTool: Cobertura
       summaryFileLocation: code-coverage/coverage.xml
 ``` 
 
  You can access the artifacts in the pipeline summary window:
 
-- To download the PDF test results report, follow the **published** link. 
-- To view the JUnit test results report, open the **Tests** tab.
-- To view the Cobertura code coverage report, open the **Code Coverage** tab.
+- To download the PDF test report, follow the **1 published** link. 
```

Comment:
Is the "1" in the link due to changes in the `PublishCodeCoverageResults` task or is it a typo?